src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/FocusManagerDirectionTests/UnoSamples_Tests.FocusManagerDirection.cs

```diff
@@ -0,0 +1,244 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml.FocusManagerDirectionTests
+{
+	public partial class UnoSamples_Tests : SampleControlUITestBase
```

Comment:
Those tests are really repetitive... Consider refactoring them a bit to improve readability:
```c#
public void FocusManager_FocusDirection_Next_Validation()
{
	Run("Uno.UI.Samples.Content.UITests.FocusManager.FocusManager_FocusDirection");

	var firstTextBox = _app.Marked("FirstTextBox");
	var secondTextBox = _app.Marked("SecondTextBox");
	var thirdTextBox = _app.Marked("ThirdTextBox");
	var nextButton = _app.Marked("NextButton");

	_app.Tap(firstTextBox);
	ChangeFocusAndAssertBeforeAfter(_app, app => app.Tap(nextButton), secondTextBox, "2", "Second text box got focus");
	ChangeFocusAndAssertBeforeAfter(_app, app => app.Tap(nextButton), thirdTextBox, "3", "Third text box got focus");
}

private void ChangeFocusAndAssertBeforeAfter(IApp app, Action<IApp> changeFocus, QueryEx target, string initialText, string finalText)
{
	// Focus target
	changeFocus(app);

	// Assert initial state
	Assert.AreEqual(initialText, target.GetDependencyPropertyValue("Text")?.ToString());

	// Update text content
	_app.ClearText();
	_app.EnterText("finalText");

	// Assert final state
	Assert.AreEqual(finalText, target.GetDependencyPropertyValue("Text")?.ToString());
}
```